k8s/namespaces/ts/ts-translation-service-int/demo.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ts-translation-service-int
+spec:
+  values:
+    java:
+      image: hmctspublic.azurecr.io/ts/translation-service:prod-00a4d27-20220530161141 #{"$imagepolicy": "flux-system:ts-translation-service"}
```

Comment:
is demo to use teh demo image policy you are adding ?

Reply:
No I assume -int will be diff to OG demo